client/tokenized-express-checkout/index.js

```diff
@@ -360,48 +358,13 @@ jQuery( ( $ ) => {
 						expressCheckoutButtonUi.blockButton();
 
 						cachedCartData = await fetchNewCartData();
-						// checking if items needed shipping, before assigning new cart data.
-						const didItemsNeedShipping = options.requestShipping;
-
-						/**
-						 * If the customer aborted the payment request, we need to re init the payment request button to ensure the shipping
-						 * options are re-fetched. If the customer didn't abort the payment request, and the product's shipping status is
-						 * consistent, we can simply update the payment request button with the new total and display items.
-						 */
-						if (
-							! wcpayECE.paymentAborted &&
-							didItemsNeedShipping ===
-								cachedCartData.needs_shipping
-						) {
-							elements.update( {
-								total: {
-									label: getExpressCheckoutData(
-										'total_label'
-									),
-									amount: transformPrice(
-										parseInt(
-											cachedCartData.totals.total_price,
-											10
-										) -
-											parseInt(
-												cachedCartData.totals
-													.total_refund || 0,
-												10
-											),
-										cachedCartData.totals
-									),
-								},
-								displayItems: transformCartDataForDisplayItems(
-									cachedCartData
-								),
-							} );
-						} else {
-							// the cachedCartData from the Store API will be used from now on,
-							// instead of the `product` attributes.
-							wcpayExpressCheckoutParams.product = null;
 
-							await wcpayECE.init();
-						}
```

Comment:
This branching logic was inherited from the non-tokenized ECE. But I noticed that it was just causing issues with 3PD extensions that altered the product price based on user interactions (like WC Deposits).

I was also noticing console logs from Stripe, saying:
> Unrecognized elements.update() parameter: displayItems is not a recognized parameter. This may cause issues with your integration in the future.
> Unrecognized elements.update() parameter: total is not a recognized parameter. This may cause issues with your integration in the future

So I think we should just get rid of them, and re-initialize the button (instead of attempting to update it).